src/main/resources/config/application-dev.yml

```diff
@@ -37,7 +37,8 @@ spring:
       indent-output: true
   datasource:
     type: com.zaxxer.hikari.HikariDataSource
-    url: jdbc:mysql://localhost:3306/oncokb_public?useUnicode=true&characterEncoding=utf8&useSSL=false&useLegacyDatetimeCode=false&serverTimezone=UTC&createDatabaseIfNotExist=true
```

Comment:
Is there a reason why we want to change the default database name?

Reply:
nope, this was when I couldn't set up on my laptop so we tried to see if it was a database connection issue